Re-import css/css-color WPT

nt1m opened the pull request:
#### cc7b4068a20eea15711a34d0c2ee75cc059e6cd9
<pre>
Re-import css/css-color WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=261895">https://bugs.webkit.org/show_bug.cgi?id=261895</a>
rdar://115853179

Reviewed by Aditya Keerthi.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/983ffcf78c4693a9abf7f8d701406f226f20ebcb">https://github.com/web-platform-tests/wpt/commit/983ffcf78c4693a9abf7f8d701406f226f20ebcb</a>

Also re-import css/support to get new color support test file.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-getcomputedstyle.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/currentcolor-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lab-l-over-100-1.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lab-l-over-100-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hsl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hwb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hwb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-mix-out-of-gamut-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-mix-out-of-gamut.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hsl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hwb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hwb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/opacity-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/opacity-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/relative-color-out-of-gamut-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/relative-color-out-of-gamut.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-a98rgb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-a98rgb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-displayp3-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-displayp3-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hwb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hwb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lab-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lab-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lch-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lch-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklab-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklab-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklch-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklch-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-prophoto-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-prophoto-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd50-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd50-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd65-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd65-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-green-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-rec2020-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-consistency-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-002-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-002.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/support/color-testcommon.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/support/computed-testcommon.js:
(test_computed_value):
* LayoutTests/imported/w3c/web-platform-tests/css/support/interpolation-testcommon.js:
(cssTransitionsInterpolation.interpolate):
(cssTransitionAllInterpolation.interpolate):
(createInterpolationTestTargets):
* LayoutTests/imported/w3c/web-platform-tests/css/support/parsing-testcommon.js:
(test_valid_value):
(test_keyframes_name_valid): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/css/support/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-monterey/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function-expected.txt:
* LayoutTests/platform/mac-monterey/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color-expected.txt:

Canonical link: <a href="https://commits.webkit.org/268293@main">https://commits.webkit.org/268293@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/0f4b03231b3d7fc1b6134b6acfe98b631ceee40a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/19263 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/19680 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/20278 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/21153 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/18020 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/22951 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/19805 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/19675 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/19482 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/19538 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/16743 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/22020 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/16724 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/17531 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/23877 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/17786 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/17706 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/21842 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/7/builds/18294 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/15489 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/17430 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/17368 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/21786 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2353 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/31/builds/18128 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder reviewed (approved):
I think some additional rebaselines are needed based on the stress test failures.

nt1m committed:
Re-import css/css-color WPT
https://bugs.webkit.org/show_bug.cgi?id=261895
rdar://115853179

Reviewed by Aditya Keerthi.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/983ffcf78c4693a9abf7f8d701406f226f20ebcb

Also re-import css/support to get new color support test file.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/border-color-currentcolor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-getcomputedstyle.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/color-mix-currentcolor-visited.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/currentcolor-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lab-l-over-100-1.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lab-l-over-100-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/lch-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/oklab-l-over-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hsl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hwb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-hwb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-invalid-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-mix-out-of-gamut-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-mix-out-of-gamut.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hsl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hwb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-hwb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-rgb-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-rgb.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/opacity-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/opacity-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/relative-color-out-of-gamut-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/relative-color-out-of-gamut.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-a98rgb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-a98rgb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-displayp3-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-displayp3-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hsl-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hwb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-hwb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lab-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lab-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lch-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-lch-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklab-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklab-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklch-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-oklch-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-prophoto-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-prophoto-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rec2020-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-02-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-rgb-02.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd50-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd50-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd65-01-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-currentcolor-xyzd65-01.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-green-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/relative-rec2020-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-consistency-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-002-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-002.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-color/system-color-hightlights-vs-getSelection-001.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-color/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/support/color-testcommon.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/support/computed-testcommon.js:
(test_computed_value):
* LayoutTests/imported/w3c/web-platform-tests/css/support/interpolation-testcommon.js:
(cssTransitionsInterpolation.interpolate):
(cssTransitionAllInterpolation.interpolate):
(createInterpolationTestTargets):
* LayoutTests/imported/w3c/web-platform-tests/css/support/parsing-testcommon.js:
(test_valid_value):
(test_keyframes_name_valid): Deleted.
* LayoutTests/imported/w3c/web-platform-tests/css/support/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-monterey/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-color-mix-function-expected.txt:
* LayoutTests/platform/mac-monterey/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color-expected.txt:

Canonical link: https://commits.webkit.org/268293@main
